app/gui/src/project-view/components/codemirror/yCollab/index.ts

```diff
@@ -0,0 +1,56 @@
+import * as cmView from '@codemirror/view'
+import { type Awareness } from 'y-protocols/awareness.js'
+import * as Y from 'yjs'
+import { YRange } from './y-range'
+import { yRemoteSelections, yRemoteSelectionsTheme } from './y-remote-selections'
+import { YSyncConfig, ySync, ySyncAnnotation, ySyncFacet } from './y-sync'
+import {
+  YUndoManagerConfig,
+  redo,
+  undo,
+  yUndoManager,
+  yUndoManagerFacet,
+  yUndoManagerKeymap,
+} from './y-undomanager'
+export {
+  YRange,
+  YSyncConfig,
+  yRemoteSelections,
+  yRemoteSelectionsTheme,
+  ySync,
+  ySyncAnnotation,
+  ySyncFacet,
+  yUndoManagerKeymap,
+}
+
+/**
+ * @param ytext
+ * @param awareness
+ * @param undoManager Set undoManager to false to disable the undo-redo plugin
+ */
```

Comment:
Please write some summary here (we don't need empty `@param`s)

If this is yCollab copy, write what changes we've made (and license information if required).